api/helpers/is-game-full.js

```diff
@@ -15,7 +15,8 @@ module.exports = {
 
   fn: ({ game }, exits) => {
 
-    const gameIsFull = !game.status || 
+    const gameIsFull =
```

Comment:
@itsalaidbacklife @Haviles04 woukd it be better to eventually have an endpoint that returns what we need as part of the response? It's possible we still need this helper but consider this:

Something like /api/games that would return a json object of all games and each game object could have a series of keys that denotes the status; isFull, etc, bawd on the frontend's needs.

There could also be endpoints for the initial available games for the game list view like /api/games/available or even /api/game/id (though I can't recall if something like this exists since I'm on my phone).

It wouldn't devalue the use-case of this helper, but might help simplify some of the code elsewhere.

Reply:
I like this idea, @itsalaidbacklife should we write up an issue on this?

Reply:
@seriouslysean is the suggestion to make it so that all games are sent, regardless of status, and the frontend filters the ones that are joinable? Or to make it so that there is a different endpoint for getting the games that are playable vs the ones that are spectatable (atm both are served by the same endpoint but queried with separate helpers)

Reply:
I think discrete endpoints make the most sense to me so that you're siloing responses to what matters for that particular request. If throughput isn't an issue one response with a key like `gameType` or similar might be ok, too. I'm really fine with whatever as long as we're making it more clear for the engineers working on this stuff. Code clarity and separation of concerns matters a lot imo, especially for open source.